### PR TITLE
dapp: Integrate udc withdrawal

### LINKED ISSUE #1421: I want to withdraw tokens from the UDC contract
## User Story
As Chris I want to withdraw tokens from the UDC contract so that I can sell them.

## Acceptance criteria
- Needs the notification area to be in place
- There is an event which helps
- There is a withdraw and deposit button (perhaps in the UDC screen)

## Tasks
- [ ] Design (Pre-Iteration design)

### PR DESCRIPTION
**Pull request by kelsos**
**Thank you for submitting this pull request :)**

Fixes #1421  

**Short description**

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Open the dApp
2. Go to the account page
3. Press the withdraw icon
4. Enter an amount
5. Wait.

### CODE REVIEW

**Comment by andrevmatos** [comment]
Looks like failures are not handled properly
I didn't have ETH on my subkey account when tried to withdraw, the failure action was properly emitted, but dialog stayed spinning, no error shown, no hidding:
image
I'm going to transfer ETH to check workflow in the happy case.

**Comment by kelsos** [comment]
Ok let me check

File: raiden-dapp/src/components/dialogs/UdcWithdrawalDialog.vue [hunk 1]
```diff
@@ -143,14 +143,19 @@ export default class UdcWithdrawalDialog extends Vue {
       await this.$raiden.planUdcWithdraw(this.withdrawAmount);
       this.done();
     } catch (e) {
-      this.error = e.message;
+      this.error = e;
```
Comment (andrevmatos) [inline raiden-dapp/src/components/dialogs/UdcWithdrawalDialog.vue:146]: Nitpick: `error` is still declared as a `string`, now it could be `any`

Comment (kelsos) [reply raiden-dapp/src/components/dialogs/UdcWithdrawalDialog.vue:146]: Thanks fixing it :+1:

Comment (kelsos) [reply raiden-dapp/src/components/dialogs/UdcWithdrawalDialog.vue:146]: Thanks fixed it :+1:

**Comment by kelsos** [comment]
> lgtm, tested here and working fine. Would we push the notification at some point, or this is it for the scope of this PR? Thanks

I think we should do that as part of https://github.com/raiden-network/light-client/pull/1648.
Let's coordinate with @taleldayekh to get it ready

**Pull request merged**